api-description/forms/pk/get.json

```diff
@@ -0,0 +1,54 @@
+{
+    "description": "A detail view for a form",
+    "url": "http://localhost:8000/forms/1",
+    "OK": {
+        "status": 200,
+        "response_data": {
+            "name": "How have you been using the site?",
+            "url": "http://localhost:8000/forms/1",
+            "fieldsets": [
+                {
+                    "name": "Free text field",
+                    "fields": [{
+                        "id": 1,
+                        "name": "How did you discover the site?",
+                        "help_text": "Search engine, friend...",
+                        "field_type": "free_text"
+                    }]
+                },
+                {
+                    "name": "Numeric fields",
+                    "fields": [{
+                        "id": 2,
+                        "name": "What is answer to the ultimate question of life, the universe, and everything?",
+                        "help_text": null,
+                        "field_type": "number"
+                    },
```

Comment:
Yes, but is should have double indentation since it is a dict in an object:
```
                   "fields": [
                         {
                                 "id": 2,
                                 "name": "What is answer to the ultimate question of life, the universe, and everything?",
                                 "help_text": null,
                                 "field_type": "number"
                           },
                           {
                                     .....
                           }
                 ]
```